consensus/consensus-types/src/experimental/commit_proposal.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::common::{Author, Round};
+use anyhow::Context;
+use diem_crypto::ed25519::Ed25519Signature;
+use std::fmt::{Debug, Display, Formatter};
+use serde::{Deserialize, Serialize};
+use short_hex_str::AsShortHexStr;
+use diem_types::{
+    ledger_info::LedgerInfo, validator_signer::ValidatorSigner,
+    validator_verifier::ValidatorVerifier,
+};
+
+
+#[derive(Deserialize, Serialize, Clone, PartialEq, Eq)]
+pub struct CommitProposal {
```

Comment:
this is more like a Vote than a Proposal, probably CommitVote

Reply:
Resolved